src/java/us/kbase/mobu/compiler/JavaTypeGenerator.java

```diff
@@ -375,7 +375,7 @@ private static void generateClientClass(JavaData data, FileSaver srcOutDir,
 					break;
 				}
 			}
-            boolean anyAsync = asyncVersion != null || anyAsync(module);
+            boolean anyAsync = asyncVersion != null;
```

Comment:
This method is 500 or so lines long, so the next 3 anyAsync bools are coming from here